src/rlgl.h

```diff
@@ -1725,6 +1726,17 @@ void rlEnableFramebuffer(unsigned int id)
 #endif
 }
 
+// return the active render texture (fbo)
+unsigned int rlGetActiveFramebuffer(void)
+{
+#if (defined(GRAPHICS_API_OPENGL_33) || defined(GRAPHICS_API_OPENGL_ES3)) && defined(RLGL_RENDER_TEXTURES_HINT)
```

Comment:
It's not really a problem but in this case if it's OPENGL 33+ or ES3
then you have:
```
return fboId;
return 0;
```
you should use:
```
GLint fboId = 0;
#if (defined(GRAPHICS_API_OPENGL_33) || defined(GRAPHICS_API_OPENGL_ES3)) && defined(RLGL_RENDER_TEXTURES_HINT)
glGetIntegerv(GL_DRAW_FRAMEBUFFER_BINDING, &fboId);
#endif
return fboId ;
```
or:
```
#if (defined(GRAPHICS_API_OPENGL_33) || defined(GRAPHICS_API_OPENGL_ES3)) && defined(RLGL_RENDER_TEXTURES_HINT)
GLint fboId = 0;
glGetIntegerv(GL_DRAW_FRAMEBUFFER_BINDING, &fboId);
return fboId ;
#else
return 0;
#endif
```
Just a minor think, it won't change the return anyway.